apps/api/Dockerfile

```
@@ -1,49 +1,30 @@
 
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 RUN corepack enable
-
-# Set working directory
 WORKDIR /app
+
+FROM base AS builder
 COPY . .
 RUN pnpm add turbo --global
 RUN turbo prune @repo/api --docker
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)